R/OsModel.R

```diff
@@ -0,0 +1,171 @@
+#' Title - TODO
+#'
+#' Description - TODO
+#'
+#' @slot stan TODO
+#' @slot templated Logical. Defines whether the OsModel object needs to be completed
+#' @export
+OsModel <- setClass(
+    Class = "OsModel",
+    representation = list(
+        stan = "StanModule",
+        templated = "logical"
+    )
+)
+
+
+#'  Log-logistic overall survival
+#'
+#' Description - Log-logistic overall survival model object creator
+#'
+#' @export
+LogLogisticOs <- setClass(
```

Comment:
Minor but perhaps worth moving this into its own file just to have separate between the classes